management/connection_test.go

```diff
@@ -354,5 +358,7 @@ yE+vPxsiUkvQHdO2fojCkY8jg70jxM+gu59tPDNbw3Uh/2Ij310FgTHsnGQMyA==
 
 		expect.Expect(t, o.GetSignInEndpoint(), "https://saml.identity/provider")
 		expect.Expect(t, o.GetTenantDomain(), "example.com")
+		expect.Expect(t, g.Metadata["foo"], "bar")
+		expect.Expect(t, g.Metadata["bat"], "baz")
```

Comment:
I'm not sure this test adds much value here, since it's just checking that the values set initially are still set. If we really wanted to check that setting metadata worked we should set it, then retrieve a new instance via a `GET` request to check against.

I'm not sure it's necessary though, since we're not doing the same for other fields, only on the `Options` field currently.

Reply:
I think you are right, I should have looked more closely at what this test is actually doing.  I will remove it.  

The ClientMetadata field on the client resource is similarly untested and that is the most similar thing in the SDK to the connection metadata so it is probably fine.